LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2016-2017, Hardy Jones 
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+ * Redistributions of source code must retain the above copyright notice,
+   this list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+ * Neither the name of  nor the names of its contributors may be used to
```

Comment:
I think this should say

> Neither the name of Hardy Jones nor the names of ...

Reply:
Right. Chalk it up to a broken license generator. I'll fix that later today and merge master in